spring-integration-redis/src/main/java/org/springframework/integration/redis/channel/SubscribableRedisChannel.java

```diff
@@ -67,15 +69,16 @@ public class SubscribableRedisChannel extends AbstractMessageChannel
 
 	private final String topicName;
 
-	private Executor taskExecutor;
+	private @Nullable Executor taskExecutor;
 
+	@SuppressWarnings("NullAway.Init")
 	private final BroadcastingDispatcher dispatcher = new BroadcastingDispatcher(true);
 
-	private RedisSerializer<?> serializer = new StringRedisSerializer();
+	private @Nullable RedisSerializer<?> serializer = new StringRedisSerializer();
```

Comment:
Hi @kimwanyoung, 😁

I'm wondering if adding `@Nullable` to this field is the right direction here.

Since the field is initialized with a non-null value and the setter enforces non-null constraints, would it make more sense to remove the unnecessary null check in the `onInit()`?

Reply:
Hi @anthologia,

Thank you for the feedback! You're absolutely correct. The `@Nullable` annotations on the `serializer` and `messageConverter` fields in `SubscribableRedisChannel.java` were unnecessary since they are always initialized and never intended to be null. I will remove these annotations and clean up the related unnecessary null-safety code in the next commit.